Client/Coordinators/Launch/LaunchCoordinator.swift

```diff
@@ -115,7 +115,11 @@ class LaunchCoordinator: BaseCoordinator,
         defaultOnboardingViewController.preferredContentSize = CGSize(
             width: ViewControllerConsts.PreferredSize.DBOnboardingViewController.width,
             height: ViewControllerConsts.PreferredSize.DBOnboardingViewController.height)
-        defaultOnboardingViewController.modalPresentationStyle = .formSheet
+        if UIDevice.current.userInterfaceIdiom == .phone {
```

Comment:
might be better to use Swift's conditional operator 
'defaultOnboardingViewController.modalPresentationStyle = (UIDevice.current.userInterfaceIdiom == .phone) ? .fullScreen : .formSheet'